pyquil/quil.py

```diff
@@ -488,7 +489,7 @@ def out(self):
             [''],
         ))
 
-    def get_qubits(self, indices=True):
+    def get_qubits(self, indices: bool = True) -> Set[int]:
```

Comment:
return type is `Set[QubitDesignator]`

Reply:
also the `isinstance` check below should probably include `ResetQubit` if you're feeling adventurous.

Reply:
I'm back to removing `ResetQubit` from the `isinstance` check because it is out of the scope of this PR.

Reply:
Fair enough